amr-wind/equation_systems/vof/split_advection.H

```diff
@@ -222,50 +140,60 @@ AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE void balance_eulerian_fluxes(
         if (bclo == amrex::BCType::ext_dir || bclo == amrex::BCType::hoextrap) {
```

Comment:
Can you separate `amrex::BCType::ext_dir` and `amrex::BCType::hoextrap`. The former should be the Dirichlet BC we use for example In wave generation while the second should be a free slip condition